.travis.yml

```diff
@@ -22,11 +21,15 @@ env:
         - EXTRA_PIP_FLAGS="--find-links=$EXTRA_WHEELS"
         - PRE_PIP_FLAGS="--pre $EXTRA_PIP_FLAGS --find-links $PRE_WHEELS"
 python:
-    - 3.4
+    - 2.7
```

Comment:
Why the extra 2.7 test?

Reply:
I think it looked like we were missing the `INSTALL_TYPE="setup"` from the `INSTALL_TYPE` block. But I guess all of them are hitting that, so I'll drop it.